src/Command/GenerateObjectAclCommand.php

```diff
@@ -60,7 +60,7 @@ class GenerateObjectAclCommand extends QuestionableCommand
     /**
      * @param RegistryInterface|ManagerRegistry|null $registry
      */
-    public function __construct(Pool $pool, array $aclObjectManipulators, $registry = null)
+    public function __construct(Pool $pool, array $aclObjectManipulators, ?ManagerRegistry $registry = null)
```

Comment:
You should remove the phpdoc and remove the useless check/deprecation in the method then.